inference-engine/scripts/tbb_retro_check.sh

```diff
@@ -0,0 +1,95 @@
+echo "Check OpenVINO binary with different version of TBB"
+
+ARCH=mac #osx
+#ARCH=lin
+#ARCH=win
+
+#declare -a TAR_EXT
+TAR_EXT=(  ["lin"]="tgz" ["win"]="zip" ["mac"]="tgz" )
+
+tbb_base_url=https://github.com/oneapi-src/oneTBB/releases/download
+
+tbb_package_map="
+2020_2   v2020.2/tbb-2020.2-XXX.EXT
```

Comment:
What will happen if user will try to specify path for newer version of TBB (e.g. 2021.1) behavior? Should we update this list each time new TBB is launched?

Reply:
Helper script. I've decided to exclude that from patch.